evap/evaluation/tests/test_models.py

```diff
@@ -239,6 +239,21 @@ def test_hidden_text_answers_get_deleted_on_publish(self):
         self.assertEqual(course.textanswer_set.count(), 2)
         self.assertFalse(TextAnswer.objects.filter(answer="hidden").exists())
 
+    def test_original_text_answers_get_deleted_on_publish(self):
+        student = mommy.make(UserProfile)
+        student2 = mommy.make(UserProfile)
+        course = mommy.make(Course, state='reviewed', participants=[student, student2], voters=[student, student2])
+        questionnaire = mommy.make(Questionnaire, type=Questionnaire.TOP)
+        question = mommy.make(Question, type="T", questionnaire=questionnaire)
+        course.general_contribution.questionnaires.set([questionnaire])
+        mommy.make(TextAnswer, question=question, contribution=course.general_contribution, answer="published answer", original_answer="original answer", state=TextAnswer.PUBLISHED)
+
+        self.assertEqual(course.textanswer_set.count(), 1)
+        self.assertEqual(TextAnswer.objects.exclude(original_answer=None).count(), 1)
+        course.publish()
+        self.assertEqual(course.textanswer_set.count(), 1)
+        self.assertFalse(TextAnswer.objects.exclude(original_answer=None).exists())
```

Comment:
the code says you want a textanswer that does not have `original_answer` not set to not exist. i don't even and i'd like you to please help